addons/web/static/src/views/widgets/ribbon/ribbon.scss

```
@@ -38,11 +23,10 @@
 
     &-top-right {
         margin-top: -$o-sheet-vpadding;
```

The component is supposed to be rendered also outside `formView`'s...
does this value works for other views too? I would be surprised.
Investigate this, eventually I propose to use css variables, something like:

```suggestion
        margin-top: var(--ribbon-margin-top, #{-$o-sheet-vpadding});
        
        . o_kanban_record & {
            --ribbon-margin-top: #{$o-kanban-inside-vgutter};
        }
```